cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -343,31 +343,38 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             event.preventDefault();
             this.$('#id_time_limit_div').hide();
             this.$('.exam-review-rules-list-fields').hide();
+            this.$('.timed-exam-options-list-fields').hide();
             this.$('#id_time_limit').val('00:00');
         },
-        selectSpecialExam: function (showRulesField) {
+        selectSpecialExam: function (additionalOptions) {
```

Comment:
There should be documentation about what the allowed values are. The current hyphenated names seem odd to me-- if you stick with this approach, I would suggest "showRulesField" and "showHideAfterDue" (these values only control whether or not the field is shown, not the selected value of them).

Two different boolean values would be more explicit, and also a more standard pattern.